.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+on: push
+
+jobs:
+  setup-linkcheck-build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Checkout sub modules
+      run: |
+        git submodule init && git submodule update
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo bash setup.sh --quiet-mode
+
+    - name: add-git-credentials
+      run: |
+        git config --global user.name "circle-ci"
+        git config --global user.email "circle-ci@nomail.com"
```

Comment:
Do you need this? For what?

Reply:
Nope, this has been removed in the current iteration. Instead the options in the corresponding github action is used.